Formula/vde.rb

```diff
@@ -23,17 +18,18 @@ class Vde < Formula
     sha256 x86_64_linux:   "d0ecff46c013cef96a1a32d6fd45d415a32dbd300932d2eb352f969445ce251c"
   end
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
-    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
-  end
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "wolfssl"
```

Comment:
WolfSSL is an odd dependency. I take it we can't use OpenSSL?

It looks like an optional dependency, though. What features does this enable?

Reply:
It enables `vde_cryptcab`, which is an encrypted variant of VDE cables. As far as I know, it requires OpenSSL.

Reply:
By "requires OpenSSL", do you mean "requires wolfSSL"? Would `qemu` make use of `vde_cryptcab`? I'd really like to avoid adding dependencies for features that would see limited use.

Reply:
Oops, mixed up my words there. I meant “doesn’t work with OpenSSL.”

A cursory search of the Debian APT repos reveals that they don’t bother to include `vde_cryptcab` with the base [`vde2`](https://packages.debian.org/bullseye/vde2) package, so I’ll go ahead and remove it here since it doesn’t seem to see much use.